parsl/dataflow/dflow.py

```diff
@@ -646,8 +643,17 @@ def submit(self, func, *args, executors='all', fn_hash=None, cache=False, **kwar
             raise ValueError("Task {} supplied invalid type for executors: {}".format(task_id, type(executors)))
         executor = random.choice(choices)
 
+        task_stdout = kwargs.get('stdout')
```

Comment:
Will putting this before [this block](https://github.com/Parsl/parsl/blob/master/parsl/dataflow/dflow.py#L652-L664) break the functionality of `parsl.AUTO_LOGNAME`?

Reply:
Probably yes

Reply:
The test introduced in #1182 shows this breakage

Reply:
#1183 is an attempt to make AppFuture's property handling a bit cleaner, driven by this problem. This PR (#1180) will need updating to accomodate that if it is merged.